config/configmap/inferenceservice.yaml

```diff
@@ -19,27 +19,43 @@ data:
                "1.13.0-gpu",
                "1.14.0",
                "1.14.0-gpu"       
+            ],
+            "supportedFrameworks": [
+              "tensorflow"
             ]
         },
         "onnx": {
             "image": "mcr.microsoft.com/onnxruntime/server",
             "defaultImageVersion": "v0.5.1",
             "allowedImageVersions": [
                "v0.5.1"
+            ],
```

Comment:
@yuzisun and @ellistarn I extended the inferenceservice configMap so each model server can advertise the ML frameworks it supports. I know onnx supports a lot more, I put popular ones here for now.

Reply:
I do not think onnx currently support multi model serving, @rakelkar can confirm.

Reply:
Yeah, I didn't find any multi-model serving information from onnx's documentation as well. But this list only advertise the supported frameworks. I added one more boolean field "multiModelServer" to indicate if this server can support multiple models.